chia/cmds/data.py

```diff
@@ -51,14 +51,15 @@ def create_changelist_option() -> Callable[[FC], FC]:
     )
 
 
-def create_key_option() -> Callable[[FC], FC]:
+def create_key_option(**kwargs: bool) -> Callable[[FC], FC]:
```

Comment:
This encourages arbitrary variation which is counter to the purpose of this function.  Also, not everything else is `bool` I don't think.  I would rather see explicit parameters for the options that have been considered.  This also makes it clear here what the variations are when considering more.

As mentioned, perhaps the name should become plural when multiple is set.